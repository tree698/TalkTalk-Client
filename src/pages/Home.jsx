import React from 'react';
import { Outlet } from 'react-router-dom';
import Navbar from '../components/Navbar';
import Footer from '../components/Footer';
import ActionBtn from '../components/ActionBtn';

export default function Home() {
  return (
    <section className="w-full h-full flex flex-col">
      <Navbar />
      <ActionBtn />
      <Outlet />
      <Footer />
    </section>
  );
}
