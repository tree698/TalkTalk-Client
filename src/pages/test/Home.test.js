import renderer from 'react-test-renderer';
import Home from '../Home';

jest.mock('../../components/Navbar');
jest.mock('../../components/ActionBtn');
jest.mock('../../components/Footer');

describe('Home', () => {
  it('renders correctly', () => {
    const component = renderer.create(<Home />);
    expect(component.toJSON()).toMatchSnapshot();
  });
});
