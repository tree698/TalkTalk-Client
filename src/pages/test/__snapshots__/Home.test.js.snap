// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders correctly 1`] = `
<section
  className="flex flex-col h-full"
>
  <div
    className="w-full fixed top-0 left-0 bg-white"
  />
  <div
    className="w-full flex-1 md:max-w-3xl lg:max-w-4xl mx-auto"
  />
</section>
`;
